packages/manager/modules/cdn-dedicated/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ovh-ux/manager-cdn-dedicated",
```

Comment:
No standalone app created with the module ?

Reply:
"Standalonisation" is done in an other PR, work is in progress ;-)